node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeadmin/NodeAdminStateUpdater.java

```diff
@@ -47,14 +59,18 @@
     private volatile State currentState = SUSPENDED_NODE_ADMIN;
 
     public NodeAdminStateUpdater(
+            NodeAgentContextFactory nodeAgentContextFactory,
             NodeRepository nodeRepository,
             Orchestrator orchestrator,
             NodeAdmin nodeAdmin,
-            HostName hostHostname) {
+            HostName hostHostname,
+            Clock clock) {
+        this.nodeAgentContextFactory = nodeAgentContextFactory;
         this.nodeRepository = nodeRepository;
         this.orchestrator = orchestrator;
         this.nodeAdmin = nodeAdmin;
         this.hostHostname = hostHostname.value();
+        this.cachedAclSupplier = new CachedSupplier<>(clock, Duration.ofSeconds(115), () -> nodeRepository.getAcls(this.hostHostname));
```

Comment:
Why 115 and not the previous 2m?

Reply:
NASU will be called _roughly_ every 30 sec, if we set cache expiry at 2m, then that would mean the cache could be used for 4 or 5 times depending on the exact timings. Setting it to 115s makes it much more likely that we stick to the previous schedule...